web/src/composebox_typeahead.ts

```diff
@@ -335,7 +335,7 @@ function handle_keydown(
             target_sel = `#${CSS.escape(target_id)}`;
         }
 
-        const on_topic = target_sel === "input#stream_message_recipient_topic";
+        const on_topic = target_sel === "#stream_message_recipient_topic";
```

Comment:
This change seems counterproductive. This `input` part here is a type-safety strategy to assert clearly it's an `input` element.

Reply:
Wouldn't **on_topic** always be false cause **target_sel** will never have that **input**?